src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1956,4 +1956,10 @@ If the "H" format specifier is used without other custom format specifiers, it's
   <data name="Create_and_assign_remaining_as_properties" xml:space="preserve">
     <value>Create and assign remaining as properties</value>
   </data>
+  <data name="Example" xml:space="preserve">
+    <value>Example:</value>
+  </data>
+  <data name="Examples" xml:space="preserve">
+    <value>Examples:</value>
+  </data>
```

Comment:
You may want to add some notes for localization here; not all languages have a singular vs. plural breakdown like this. Usually in that case I'm OK with just both getting translated to whatever the language convention is for Example(s) but the localizer might not know what to do here.

I recall @KirillOsenkov flagging me doing exactly this ages ago... :smile:

Reply:
Humanizer also has support for pluralization like this that we might be able to leverage.

Reply:
added localizer notes.